evalml/tests/automl_tests/test_automl_search_classification.py

```diff
@@ -224,7 +226,7 @@ def test_optimizable_threshold_enabled(mock_fit, mock_score, mock_predict_proba,
     mock_score.assert_called()
     mock_predict_proba.assert_called()
     mock_optimize_threshold.assert_called()
-    assert automl.best_pipeline.threshold is None
+    assert automl.best_pipeline.threshold is not None
```

Comment:
@bchen1116 ah, good call. Will the best pipeline have bin class threshold optimized? We should do what is necessary to make that true.

Reply:
I see. I don't think it is currently, but I can fix that. Thanks!